script/deploy/facets/UpdateSymbiosisFacet.s.sol

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.17;
+
+import { UpdateScriptBase } from "./utils/UpdateScriptBase.sol";
+import { stdJson } from "forge-std/StdJson.sol";
+import { DiamondCutFacet, IDiamondCut } from "lifi/Facets/DiamondCutFacet.sol";
+import { SymbiosisFacet } from "lifi/Facets/SymbiosisFacet.sol";
+
+contract DeployScript is UpdateScriptBase {
+    using stdJson for string;
+
+    function run() public returns (address[] memory facets, bytes memory cutData) {
+        address facet = json.readAddress(".SymbiosisFacet");
```

Comment:
you are reading from the json variable before parsing the file into it. Unless there's some hidden meaning to it, I doubt this will work